Example/Backpack/Views/BPKTableViewSelectableCell.swift

```diff
@@ -41,7 +41,9 @@ class BPKTableViewSelectableCell: UITableViewCell {
     }
 
     override func didMoveToWindow() {
-        tickIcon.tintColor = Theme.primaryColor(for: self)
+        if self.window != nil {
```

Comment:
I'm not sure why this check is needed. It appears that they are only outside the window when I change theme...

![2019-05-08 16 30 56](https://user-images.githubusercontent.com/30267516/57387925-21b16480-71af-11e9-9b53-c2ebad869412.gif)

Reply:
Overriding `didMoveToSuperview` has the same downside....